src/test/java/pages/ForgotPasswordPage.java

```diff
@@ -0,0 +1,33 @@
+package pages;
+
+import managers.FileReaderManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import utility.BaseClass;
+
+public class ForgotPasswordPage extends BaseClass {
+
+    WebDriver driver;
+
+    public ForgotPasswordPage(WebDriver driver){
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+
+    }
+
+    @FindBy(id = "email")
+    private WebElement email;
+
+    @FindBy(id = "form_submit")
+    private WebElement retrievePassword;
+
+    @FindBy(tagName = "h1")
+    public WebElement error_message;
+
+    public void goToForgotPasswordPage(){
+        driver.get(FileReaderManager.getInstance().getConfigFileReader().getApplicationURL().concat("forgot_password"));
```

Comment:
Try creating new "config" helper class to read the config file - that way you will have driver.get(ConfigHelper.get("forgot_password")); which is more readible

Reply:
I did it in a bit of a roundabout way, using the classes I had available already, just rerouted them through ConfigHelper class.